peertalk.xcodeproj/project.pbxproj

```diff
@@ -44,15 +44,16 @@
 		6A88FA63150D61DE00FC3647 /* PTUSBHub.m in Sources */ = {isa = PBXBuildFile; fileRef = 6A88FA5F150D61DE00FC3647 /* PTUSBHub.m */; };
 		6ACFD2D6151D36220081ACF5 /* PTChannel.h in Headers */ = {isa = PBXBuildFile; fileRef = 6ACFD2D4151D36220081ACF5 /* PTChannel.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		6ACFD2D7151D36220081ACF5 /* PTChannel.m in Sources */ = {isa = PBXBuildFile; fileRef = 6ACFD2D5151D36220081ACF5 /* PTChannel.m */; };
+		6E1CFFCB24DA061900D8CCEB /* Peertalk.h in Headers */ = {isa = PBXBuildFile; fileRef = 6E1CFFCA24DA061900D8CCEB /* Peertalk.h */; };
+		6E1CFFCC24DA061900D8CCEB /* Peertalk.h in Headers */ = {isa = PBXBuildFile; fileRef = 6E1CFFCA24DA061900D8CCEB /* Peertalk.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
I'm surprised that both headers (with equal identifiers) are listed in the `Headers` group, each with different build settings. But 3 have been removed further down the file 🤷‍♂️